eva-server/src/main/java/uk/ac/ebi/eva/server/ws/ArchiveWSServer.java

```diff
@@ -110,8 +115,14 @@ public QueryResponse getStudies(@RequestParam(name = "species", required = false
     @RequestMapping(value = "/studies/list", method = RequestMethod.GET)
     public QueryResponse getBrowsableStudies(@RequestParam("species") String species)
             throws IllegalOpenCGACredentialsException, IOException {
-        StudyDBAdaptor studyMongoDbAdaptor = DBAdaptorConnector.getStudyDBAdaptor(species);
-        return setQueryResponse(studyMongoDbAdaptor.listStudies());
+        MultiMongoDbFactory.setDatabaseNameForCurrentThread(DBAdaptorConnector.getDBName(species));
+        List<VariantStudySummary> uniqueStudies = variantStudySummaryRepository.findBy();
+
+        QueryResult<VariantStudySummary> result = new QueryResult<>();
```

Comment:
With these usages in addition to those in [RegionWSServer](https://github.com/EBIvariation/eva-ws/blob/develop/eva-server/src/main/java/uk/ac/ebi/eva/server/ws/RegionWSServer.java) this `QueryResult` initialization is getting very redundant. Could this small code block be extracted to a method in the parent class?